packages/core/src/app/order/OrderSummary.tsx

```diff
@@ -41,6 +119,28 @@ const OrderSummary: FunctionComponent<OrderSummaryProps & OrderSummarySubtotalsP
                 {additionalLineItems}
             </OrderSummarySection>
 
+            {loyaltiPoints?.balance?.available && (
+              <OrderSummarySection>
+                <div className='reward-section'>
+                  <h3 className="loyalty-discount optimizedCheckout-headingSecondary">
+                    Loyalty Discount
+                  </h3>
+                  <div>
+                    <span className='cart-priceItem-value'>{loyaltiPoints?.balance?.available}</span> off available.
+                    <a
+                      onClick={() => {
+                        if (applyGiftCertificate) {
```

Comment:
Should we show the "Loyalty Points" section at all, if this function isn't passed as a prop?

Reply:
No, I will add this check as well for the loyalty point section